.ci/github-upload-windows.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh -ex
+
+ARTIFACT_DIR="$BUILD_ARTIFACTSTAGINGDIRECTORY"
+generate_post_data()
+{
+    body=`cat GitHubReleaseMessage.txt`
```

Comment:
Prefer `$()` to ``, per [SC2006](https://github.com/koalaman/shellcheck/wiki/SC2006)